lib/bolt/outputter/human.rb

```diff
@@ -14,12 +14,14 @@ class Human < Bolt::Outputter
 
       def print_head; end
 
-      def initialize(color, verbose, trace, stream = $stdout)
+      def initialize(color, verbose, trace, spin, stream = $stdout)
         super
         # Plans and without_default_logging() calls can both be nested, so we
         # track each of them with a "stack" consisting of an integer.
         @plan_depth = 0
         @disable_depth = 0
+        @dots = %w[⠋ ⠙ ⠹ ⠸ ⠼ ⠴ ⠦ ⠧ ⠇ ⠏]
```

Comment:
Is this going to be configurable? If not, it should just be removed for now.

Reply:
Yeah - I ended up going with the pinwheel for now but it'd be fun to add more later :)